thrust/device_vector.h

```diff
@@ -69,6 +69,9 @@ template<typename T, typename Alloc = thrust::device_malloc_allocator<T> >
     device_vector(void)
       :Parent() {}
 
+    __host__
```

Comment:
This new destructor needs a Doxygen comment. You can use the Doxygen for the default constructor for an example.